src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -777,3 +777,41 @@
             :onClick #(swap! state assoc :collapsed? (not (:collapsed? @state)))})
           body]
          body)))})
+
+(react/defc FilterGroupBar
```

Comment:
could this `FilterGroupBar` be aligned with the FilterGroupBar in table_utils.cljs? There's a lot of duplicated code between the two.